src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -189,10 +189,15 @@ public final class Guardrails implements GuardrailsMBean
      */
     public static final EnableFlag bulkLoadEnabled =
     new EnableFlag("bulk_load_enabled",
-                   null,
+                   "Bulk loading of SSTables might potentially destabilize the node.",
                    state -> CONFIG_PROVIDER.getOrCreate(state).getBulkLoadEnabled(),
                    "Bulk loading of SSTables");
 
+    static
```

Comment:
I dont like this. Can't you just call that method as you instantiate `bulkLoadEnabled`? Like this:

    new EnableFlag("bulk_load_enabled",
                   null,
                   "Bulk loading of SSTables might potentially destabilize the node.",
                   state -> CONFIG_PROVIDER.getOrCreate(state).getBulkLoadEnabled(),
                   "Bulk loading of SSTables").throwOnNullClientState(true);

Reply:
+1, although it would require a trivial cast to `EnableFlag` since `Guardrail#throwOnNullClientState` returns a `Guardrail`:
```java
public static final EnableFlag bulkLoadEnabled =
(EnableFlag) new EnableFlag("bulk_load_enabled",
                            "Bulk loading of SSTables might potentially destabilize the node.",
                            state -> CONFIG_PROVIDER.getOrCreate(state).getBulkLoadEnabled(),
                            "Bulk loading of SSTables").throwOnNullClientState(true);
```